docs/samples/tensorflow/README.md

```diff
@@ -0,0 +1,60 @@
+
+# Predict on a KFService using Tensorflow
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. Your cluster's Istio Egresss gateway must [allow Google Cloud Storage](https://knative.dev/docs/serving/outbound-network-access/)
+
+## Create the KFService
+Apply the CRD
+```
+kubectl apply -f tensorflow.yaml 
+```
+
+Expected Output
+```
+$ kfservice.serving.kubeflow.org/flowers-sample configured
+```
+
+## Run a prediction
+
+```
+MODEL_NAME=flowers-sample
+INPUT_PATH=@./input.json
+CLUSTER_IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+
+curl -v -H "Host: flowers-sample.default.svc.cluster.local" http://$CLUSTER_IP/v1/models/$MODEL_NAME:predict -d $INPUT_PATH
+```
+Expected Output
+```
+*   Trying 34.83.190.188...
```

Comment:
You may want to get rid of ip addresses here

Reply:
Eh I'm not concerned. These are publically assignable GKE cluster IPs. I don't even own this anymore.